ports/esp32s2/boards/lilygo_ttgo_t8_s2_st7789/pins.c

```diff
@@ -41,13 +42,13 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_SD_CS), MP_ROM_PTR(&pin_GPIO10) },
 
     // 1.14 inch LCD ST7789
-    { MP_ROM_QSTR(MP_QSTR_LCD_MISO), MP_ROM_PTR(&pin_GPIO4) },
```

Comment:
Please leave this. There is a chance folks will want to re-initialize the display themselves. (You can have two names map to the same pin here.)

Reply:
Does it makes sense if MISO is not connected to IO4? according to @skieast and the schematics on LilyGo repo it isn't connected.

Reply:
@skieast but as I'm looking on the schematic now...what about IO37? how can I test if IO37 is MISO? 
<img width="381" alt="image" src="https://user-images.githubusercontent.com/77211708/107087452-c4bb6600-67fb-11eb-91f4-9c340f5e4b4c.png">

Reply:
Unfortunately their labels don't match the actual function.  As their examples don't match the hardware. I spent a few hours trying to read the ST7789 registers with no success, trying all the various pins. 
Looking at he OLED end they label everyything as if it was I2C and not SPI
![image](https://user-images.githubusercontent.com/25512/107088434-c6f8d280-67b1-11eb-85c0-bd595910229f.png)

I rang out the various pins using a meter and there is no MISO on the display.  They use the same display on their GD32 board and it more clearly shows the connections.

Reply:
![image](https://user-images.githubusercontent.com/25512/107088689-31aa0e00-67b2-11eb-8b13-e0838d186bb0.png)
Here are the connections to the same display on the GD32 board

Reply:
ok, then I would prefer to leave the code as it is. no MISO pin for the display.

Reply:
Thanks for running this down!